client/src/containers/DistrictAlertsPage.tsx

```diff
@@ -240,6 +245,27 @@ const DistrictCreation = withI18n()((props: withI18nProps) => {
           onClick={saveSelections}
         />
       </div>
+      <Modal
+        showModal={showEmailModal}
+        width={40}
+        onClose={() => setShowEmailModal(false)}
+        className="area-alerts-modal"
+        newStyle={true}
+      >
+        <div className="modal__content">
+          <h3>
+            <Trans>Sample Area Alert Email</Trans>
+          </h3>
+
+          <figure>
+            <img
+              src="/area-alert-example-email.png"
+              alt="Sample image of area alert email showing building data and risk indicators"
```

Comment:
Seems like our CI test/lint has a problem with including the word "image" in alt text for <img>
```suggestion
              alt="Sample of area alert email showing building data and risk indicators"
```

Reply:
thanks for catching & fixing this!